spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/DataFlowServerConfiguration.java

```diff
@@ -307,6 +302,18 @@ public TaskRepositoryInitializer taskRepositoryInitializerForDefaultDB(DataSourc
 			return taskRepositoryInitializer;
 		}
 
+		@Bean
+		public DefinitionRepositoryInitializer definitionRepositoryInitializerForDefaultDB(DataSource dataSource, Server server) {
```

Comment:
What is the reasoning behind having two separate `definitionRepositoryInitializerForDefaultDB` beans? I guess we don't use the H2 `Server` here?